Compound_Interest/Compound_Interest.c

```diff
@@ -0,0 +1,26 @@
+/*
+    Compound Interest = P(1 + R/100)^T
+    
+    P is principle amount
+    R is the rate and
+    T is the time span
```

Comment:
No need to add indentation here. Remember this for future PRs. Approving this for now.